bin/ghe-backup

```diff
@@ -1,12 +1,40 @@
 #!/usr/bin/env bash
-#/ Usage: ghe-backup [-v] [--version]
+#/ Usage: ghe-backup [-h] [-v] [--version]
 #/ Take snapshots of all GitHub Enterprise data, including Git repository data,
 #/ the MySQL database, instance settings, GitHub Pages data, etc.
 #/
-#/ With -v, enable verbose output and show more information about what's being
-#/ transferred.
+#/ Options:
+#/   -v                Enable verbose output.
+#/   --version         Display version information.
+#/   -h                Show this message.
+
```

Comment:
Can you please change this and other locations to use the format we use for the usage in the commands included in GitHub Enterprise. The general format we use is like this:

```
#!/bin/bash
#/ Usage: ghe-maintenance [-hqsu]
#/
#/ Set or unset the maintenance mode on GitHub Enterprise
#/
#/ OPTIONS:
#/   -h | --help      Show this message
#/   -q | --query     Query current mode
#/   -s | --set       Set the maintenance mode
#/   -u | --unset     Unset the maintenance mode
#/
```

Specific things to note:

- grouping of short options in the initial `Usage` line
- OPTIONS is capitalised
- Short & long options are shown in the OPTIONS section, where available.